Description: move the clonk binary to /usr/games
 The FHS says that game binaries should be stored in /usr/games.
Author: Philipp Kern <pkern@debian.org>

--- openclonk-5.3.1.orig/CMakeLists.txt
+++ openclonk-5.3.1/CMakeLists.txt
@@ -1404,7 +1404,8 @@ endif()
 install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/clonk.desktop DESTINATION share/applications)
 
 # Install binaries
-install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/clonk ${CMAKE_CURRENT_BINARY_DIR}/c4group DESTINATION bin/)
+install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/clonk DESTINATION games/)
+install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/c4group DESTINATION bin/)
 
 ############################################################################
 # setup_openclonk.exe
